src/main/java/org/jpeek/App.java

```diff
@@ -131,7 +131,8 @@ public App(final Path source, final Path target) {
                 new MapEntry<>("LCOM5", true),
                 new MapEntry<>("SCOM", true),
                 new MapEntry<>("NHD", true),
-                new MapEntry<>("MMAC", true)
+                new MapEntry<>("MMAC", true),
+                new MapEntry<>("OCC", true)
```

Comment:
@llorllale Why did you add OCC here? Does it have to do with metric.xsl?

Reply:
@amihaiemil fixed; this was an error on my part